se_mailbox/filelock.py

```diff
@@ -19,11 +20,10 @@ class FileLockException(Exception):
 
 class FileLock(object):
     """A file locking mechanism that has context-manager support so
-    you can use it in a with statement. This should be relatively cross
-    compatible as it doesn't rely on msvcrt or fcntl for the locking."""
+    you can use it in a with statement.
```

Comment:
Why did we change this so it uses fcntl?

Reply:
@RoxanaButuc unanswered query here.

Reply:
Changes were made based on this [comment](https://jira.solarwinds.com/browse/MMA-868?focusedCommentId=1714710&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1714710) - in order to get the package up to date with mail-core